sysinit/coqargs.ml

```diff
@@ -448,7 +448,12 @@ let build_load_path opts =
     else
       let coqenv = Boot.Env.init () in
       Coqloadpath.init_load_path ~coqenv in
-  ml_path @ opts.pre.ml_includes ,
+  let ocamlpathsep = if Sys.unix then ':' else ';' in
+  let env_ocamlpath =
```

Comment:
Didn't we say that when -boot is set, we don't use OCAMLPATH?

Reply:
That is the opposite of what I understood.
-boot is about not binding the stdlib, so it is unrelated to OCAMLPATH

Reply:
Yes, OCAMLPATH should always be used (e.g., critical on Nix). Otherwise, dependencies will not be found.